x-pack/plugins/canvas/public/components/sidebar/sidebar_content.js

```diff
@@ -21,22 +24,43 @@ const branches = [
   // no elements selected
   branch(
     ({ selectedToplevelNodes }) => !selectedToplevelNodes.length,
-    renderComponent(GlobalConfig)
+    renderComponent(() => (
+      <>
+        <GlobalConfig />
+      </>
+    ))
   ),
   // multiple elements selected
   branch(
     ({ selectedToplevelNodes }) => selectedToplevelNodes.length > 1,
-    renderComponent(MultiElementSettings)
+    renderComponent(() => (
+      <>
```

Comment:
Nit: consider breaking these out into individual `const`s above... it will make the branches easier to see together.